src/main/java/hudson/model/ExternalJob.java

```diff
@@ -122,6 +122,39 @@ public String getDisplayName() {
             return Messages.ExternalJob_displayName();
         }
 
+        /**
+         * Needed if it wants External Monitor Jobs are categorized in Jenkins 2.x.
+         *
+         * TODO: Override when the baseline is upgraded to 2.x
+         *
+         * @return A string it represents a ItemCategory identifier.
+         */
+        public String getCategoryId() {
+            return "standaloneprojects";
+        }
+
+        /**
+         * Needed if it wants External Monitor Jobs are categorized in Jenkins 2.x.
+         *
+         * TODO: Override when the baseline is upgraded to 2.x
+         *
+         * @return A string with the Item description.
+         */
+        public String getDescription() {
+            return Messages.ExternalJob_Description();
+        }
+
+        /**
+         * Needed if it wants External Monitor Jobs are categorized in Jenkins 2.x.
+         *
+         * TODO: Override when the baseline is upgraded to 2.x
+         *
+         * @return A string it represents a URL pattern to get the Item icon in different sizes.
+         */
+        public String getIconFilePathPattern() {
+            return (Jenkins.RESOURCE_PATH + "/plugin/external-monitor-job/images/:size/externaljob.png").replaceFirst("^/", "");
```

Comment:
@jglick If you agree, I prefer to use the original format, used in other plugins:

```
public String getIconFilePathPattern() {
    return "plugin/external-monitor-job/images/:size/externaljob.png";
}
```

In my opinion, more flexible.